Wglc: comments from Thomas Fossati (#108)
core-wg/href: draft-ietf-core-href.md

  When converted to URIs, these would violate the requirement that in absence of an authority, a URI's path cannot begin with two slash characters,
  and they would be indistinguishable from a URI with a shorter path and a present but empty authority component.

> * {:#naked-rootless} CRIs without authority that are rootless and do not have a pathg

Reviewer: typo `pathg`

fail gracefully in the face of malicious inputs.)

CRIs have been designed with the objective that, after the above
normalization, conversion of two distinct CRIs to URIs do
not yield the "same" URI, including equivalence under syntax-based
normalization ({{Section 6.2.2 of RFC3986@-uri}}), but not including
protocol-based normalization.

> Note that this exclusively is an objective for (absolute) CRIs, not
> for CRI references: these need to be resolved relative to a base URI,

Reviewer: ```suggestion
Note that this objective exclusively applies to (absolute) CRIs, not
to CRI references: these need to be resolved relative to a base URI,
```
